network/transport/grpc/connection.go

```diff
@@ -99,7 +100,7 @@ type conn struct {
 }
 
 func (mc *conn) Peer() transport.Peer {
-	// Populated through createConnection and verifyOrSetPeerID
+	// Populated through createConnection and setPeerID
```

Comment:
Should be `verifyOrSetPeerID` I believe (could be part of the rebase/merge)

Reply:
I renamed it